projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -219,7 +208,7 @@ export class IgxInputGroupComponent extends DisplayDensityBase
      * ```
      */
     public get type() {
-        return this._type;
+        return this._type ? this._type : this._inputGroupType ? this._inputGroupType : this._type = 'line';
```

Comment:
The same change can be done to the `IgxCombo` as well - add the token to the constructor (w\ `@Optional()` decorator). If no token is provided, fall back to the default `'box'` value, to avoid any behavioral changes.

```
private _type = null;
...
public get type(): IgxInputGroupType {
        return this._type || this._inputGroupToken || 'box';
    }

public set type(val: IgxInputGroupType) {
    this._type = val;
}

```